components/Card.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import Image from "next/image";
+
+
+
+type Props = {
+  Image: any
+  name:string
+  title:string
+};
+
+const Card: React.FC< Props > = (props) => {
+  return (
+    <div className="border border-none w-56 h-72 mx-auto ">
+        <Image src={props.Image } width={500} height={300} />
```

Comment:
Image elements must have an alt prop, either with meaningful text, or an empty string for decorative images. Do this for all `Image` tags in your code.